### PR TITLE
[FEATURE] Add Links to Partner Icons 

### LINKED ISSUE #310: [FEATURE] Add Links to Partner Icons
### Description

It's better to have links to the partner icons.

- Wemakedevs https://wemakedevs.org/
- 4C https://www.4c.rocks/

### Screenshots

Screenshot from 2023-11-06 17-28-38

### Additional information

_No response_

### PR DESCRIPTION
**Pull request by ninadvyas**
`Closes #310 ` Add Links to Partner Icons : #310

## Changes proposed

- Added links to the partner icons

## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

## Screenshots of #310 
Screenshot 2023-12-16 at 3 21 25 PM

### CODE REVIEW

File: src/components/Footer.jsx [hunk 1]
```diff
@@ -35,12 +35,14 @@ export function Footer() {
                 >
                   {group.map((company) => (
                     <li key={company.name} className="flex">
+                      <a href={company.href} target="_blank" title={company.title}>
```
Comment (xkrishguptaa) [inline src/components/Footer.jsx:38]: Please use the nextjs link tag!

**Comment by ninadvyas** [comment]
@xkrishguptaa I have made changes accordingly as you said.

File: src/components/Footer.jsx [hunk 2]
```diff
@@ -35,12 +35,14 @@ export function Footer() {
...
                   {group.map((company) => (
                     <li key={company.name} className="flex">
+                      <Link href={company.href} target="_blank" title={company.title}>
                       <Image
                         src={company.logo}
                         alt={company.name}
                         unoptimized
                         width="50"
                       />
+                      </Link>
```
Comment (xkrishguptaa) [inline src/components/Footer.jsx:45]:
```suggestion
                      <Link href={company.href} target="_blank" title={company.title}>
                      	<Image
                        	src={company.logo}
                        	alt={company.name}
                        	unoptimized
                        	width="50"
                      	/>
                      </Link>
```

**Review by xkrishguptaa — CHANGES_REQUESTED** [review]
Did you import the link tag? Because I do not see it in the imports?

**Comment by ninadvyas** [comment]
@xkrishguptaa link tag already there imported at first line of the file. Waiting for the feedback.

**Review by xkrishguptaa — APPROVED** [review]
Ah okay! sorry missed it earlier, thank you for the contribution

**Pull request merged**